media/js/base/mozilla-utils.js

```diff
@@ -50,9 +57,13 @@ if (typeof Mozilla === 'undefined') {
     // then, each key name needs to be preceded by data- as this uses data attributes
     // to work. After this, you can access all strings defined inside the
     // string_data block in JS using Mozilla.Utils.trans('keyofstring'); Thank @mkelly
-    var _$strings = $('#strings');
+    var _strings = document.getElementById('strings');
     Utils.trans = function(stringId) {
-        return _$strings.data(stringId);
+        if (_strings) {
+            return _strings.getAttribute('data-' + stringId);
```

Comment:
Hmm, this isn't quite going to work as-is.

Whilst some calls to this function [use dashes](https://github.com/mozilla/bedrock/blob/master/media/js/firefox/accounts-2018.js#L25) to separate attribute names, other calls use [camelCase](https://github.com/mozilla/bedrock/blob/bb1a65b048f130a4e1baa70a1a040eb462838c55/media/js/mozorg/contact-spaces.js#L23), which for the jQuery API (and native dataset) is totally fine, but would break here as we're assuming dashes only.

Either this function should handle both cases (more complex), or we should choose one method over the other, and update all calls to it to be the same syntax. Thoughts?

Reply:
Good catch. 🎣 

In a perfect world we handle both cases but in the name of expediency I'm tempted to just go with dashes and update the code base.

Reply:
OTOH there is a certain irony to talking about expediency in a 3 month old PR.

Reply:
Dash is way more common, converting to dashes.